docs/chainlink-keepers/compatible-contracts.md

```diff
@@ -11,28 +11,57 @@ whatsnext:
 
 ## Overview
 
-This guide explains how to make smart contracts **Keeper-compatible**. You will learn about the `KeeperCompatibleInterface` and its functions with an example contract. The guide showcases the convenience that Keepers provide to developers. To take full advantage of the Keepers automation infrastructure, read all of the documentation to understand the features of Chainlink Keepers.
+This guide explains how to make smart contracts **Keeper-compatible** by walking you through an example. You will learn about the `KeeperCompatibleInterface` and its functions with an example contract. The guide showcases the convenience that Keepers provide to developers. To take full advantage of the Keepers automation infrastructure, read all of the documentation to understand the features of Chainlink Keepers.
 
 **Table of Contents**
-
++ [Example Contract](#example-contract)
 + [Functions](#functions)
   + [`checkUpkeep` function](#checkupkeep-function)
     + [`checkData`](#checkdata)
     + [`performData`](#performdata)
   + [`performUpkeep` function](#performupkeep-function)
     + [`performData`](#performdata-1)
-+ [Example Contract](#example-contract)
+
+
+
+
+## Example Contract
+
+Let's look at the core elements of a Keepers-compatible contract, be examining a simple counter contract that will increment once a specified amount of time has passed. The Keepers Network will perform `checkUpkeep` off-chain during every block to see if `updateInterval` seconds has passed since the last timestamp. Once the specified amount of time has elapsed checkUpkeep will return true and the Keepers Network will execute `performUpkeep` on-chain to increment the counter.
```

Comment:
there's a small typo here...it currently says "be examining a simple counter...." the "**be**" should be replaced by a "**by**"

Reply:
thanks!